README.md

```diff
@@ -44,3 +44,18 @@ pip install -r requirements.txt
 # run the setup
 python multi_clone.py
 ```
+When you execute `python multi_clone.py`, you will see following prompts that guide you through setting up your node(s).
+
+1. **Terminate Existing containers:** "Do you want to kill all running containers and screen sessions of testnet nodes? (y/n) n"
+
+- Type `y` Use this option and stop all active containers or node instances. This will clean up all the older containers. Please cross check your running containers before executing this command. 
+
+2. **Custom Slot ID Deployment:** "Do you want to deploy a custom index of slot IDs (indices begin at 0, enter in the format [begin, end])? (indices/n)"
```

Comment:
elaborate on the case when `n` is entered, where the entire array of slot IDs are chosen to deploy nodes against

Reply:
Oh yes! Apologies. Missed that case. Will add and push another commit!

Reply:
Addressed